### DISCUSSION TITLE
Is there any research or consideration on how gradients effects legibility?

### DISCUSSION THREAD

**Discussion by chris480** [Myndex/SAPC-APCA#102] [THEORY]
Was thinking about a small comment from a Microsoft design talk I attended years ago, in which the speaker mentioned slight background gradients on elements improved perceived contrast for text. Examples of visual illusions were used to demonstrate changes in contrast. 

![]( https://i.imgur.com/vUlnp97.jpg )

Checker board classic

![]( https://upload.wikimedia.org/wikipedia/commons/thumb/b/be/Checker_shadow_illusion.svg/2560px-Checker_shadow_illusion.svg.png )

**Comment by Myndex** [comment]
Hi @chris480,

The checkerboard illusion is one of my favorites because it so well demonstrates the importance of local adaptation.

In this version both yellow dots are identical:

![YellowDotCheckerShadow_DLyon_PD](https://user-images.githubusercontent.com/42009457/215248687-01e232a8-a1e7-49ce-9ce8-d4187929f8b9.png)


As for gradients, there's been a lot of study about things like simultaneous contrast, But not as much regarding text,  sounds like something I need to add to my research list.

Also See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3767278/#!po=36.6667

Also Google: 
- Bartleson-Breneman Effect 
- Mach Bands
- Simultaneous contrast

**Comment by chris480** [comment]
Thanks for the reply. I'll dig into the paper things around the terms you listed, but on a cursory glance I haven't come across anything as clear cut as the algorithms presented in APCA. Maybe there is an opportunity here to codify something. If it doesn't exist, I know some interested parties that could throw resources towards doing a study.

**Comment by Myndex** [comment]
> *If it doesn't exist, I know some interested parties that could throw resources towards doing a study.*

As this is my current research focus, I'd be very interested in discussing this further. Can you email me at perceptex@myndex.com ?

**Comment by Myndex** [comment]
This should be in discussions, moving...
